test/GenerateProgrammaticPipelineFromConfig/GeneratorTest.php

```diff
@@ -77,7 +83,9 @@ public function testGeneratesExpectedPipeline()
     public function testRaisesExceptionIfConfigFileNotFound()
     {
         $configFile = vfsStream::url('project/config/config.php');
-        $this->setExpectedException(GeneratorException::class, 'not found');
+
+        $this->expectException(GeneratorException::class);
+        $this->expectExceptionMessage('not found');
```

Comment:
This seems overly verbose, compared to the original. What prompted this change, exactly?